CharlieBackend.Api/Controllers/CoursesController.cs

```diff
@@ -36,23 +37,9 @@ public CoursesController(ICourseService coursesService)
         [HttpPost]
         public async Task<ActionResult<CreateCourseDto>> PostCourse(CreateCourseDto courseDto)
         {
-            if (!ModelState.IsValid) return BadRequest();
```

Comment:
Separate task for unit-test and moving a logic from CourseCountroller into CourseService for two different PRs.
It's not good to close 2 different enchantments in one PR.

Reply:
For moving a logic from CourseCountroller into CourseService use issue #203